controllers/v1/static-links.js

```diff
@@ -0,0 +1,232 @@
+/**
+ * name : staticLinks.js
+ * author : Rakesh
+ * created-date : 28-Oct-2020
+ * Description : Static links related information.
+ */
+
+// Dependencies
+const csv = require("csvtojson");
+const staticLinksHelper = require(MODULES_BASE_PATH + "/static-links/helper")
+const FileStream = require(ROOT_PATH + "/generics/file-stream");
+
+/**
+    * StaticLinks
+    * @class
+*/
+module.exports = class StaticLinks extends Abstract {
+  constructor() {
+    super(schemas["static-links"]);
+  }
+
+  static get name() {
+    return "static-links";
+  }
+
+  /**
+  * @api {get} /kendra/api/v1/static-links/list Static Link list
+  * @apiVersion 1.0.0
+  * @apiName Static Link list
+  * @apiGroup Static Links
+  * @apiHeader {String} X-authenticated-user-token Authenticity token
+  * @apiSampleRequest /kendra/api/v1/static-links/list
+  * @apiParamExample {json} Response:
```

Comment:
@rakeshSgr - Does this not have to include appType and appName? Samiksha latest code is there for the same.

Cc -  @aman-tunerlabs